src/pages/CubeDeckPage.js

```diff
@@ -36,12 +40,21 @@ const CubeDeckPage = ({ cube, deck, canEdit, userid }) => {
             </Label>
             <Input type="select" id="viewSelect" value={seatIndex} onChange={handleChangeSeat}>
               {deck.seats.map((seat, index) => (
-                <option key={seat.userid || seat.bot} value={index}>
+                <option key={/* eslint-disable-line react/no-array-index-key */ index} value={index}>
```

Comment:
not sure, but could this be solved by something like this:

```
<option key={"seat"+index} value={index}>
```

Reply:
We use this current pattern elsewhere as well. I personally think we should remove this rule. What is the reasoning behind it exactly?

Reply:
react component keys should be unique to the item itself for dom change reasons. if you use an index, it's not unique to the object itself but its location in a list.

Reply:
Makes sense, revised

Reply:
this is a bit more information that I found...

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md